schema/alpha/containers/workspace.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The containers / workspace concept seems rather vaguely defined -- what is its purpose and does it belong in metadata?

Reply:
replied inline